src/components/box/_box.scss

```diff
@@ -1,8 +1,10 @@
 $boxBorderColor: $graySecondary;
 $boxBorderRadius: 22px;
-$boxBorderSize: rhythm(1/8);
+$boxBorderSize: rhythm(1 / 8);
```

Comment:
When we have it as `1 / 8` I do not like this style anymore... :(
But I think its ok